Modules/Core/SpatialObjects/include/itkSpatialObjectProperty.h

```diff
@@ -108,25 +134,60 @@ class ITKSpatialObjects_EXPORT SpatialObjectProperty
     return m_Name;
   }
 
+  std::string
+  Name() const
+  {
+    return m_Name;
+  }
```

Comment:
According to https://www.swig.org/Doc4.2/SWIGDocumentation.html SWIG _does_ support `const std::string &`, but honestly it's not clear to me how 🤷  It says:

> The `%naturalvar` feature was introduced in SWIG-1.3.28, prior to which it was necessary to manually apply the const reference typemaps, eg `%apply const std::string & { std::string * }`, but this example would also apply the typemaps to methods taking a std::string pointer.

And:

> The naturalvar behavior can also be turned on as a global setting via the `-naturalvar` commandline option or the module mode option, `%module(naturalvar=1)`.

Would "the naturalvar behavior" be of help to us?